### PR TITLE
Implemented timeout support. Solves #496

### PR DESCRIPTION
**Pull request by dmatora**

### CODE REVIEW

File: lib/Remote/RemoteWebDriver.php [hunk 1]
```diff
@@ -141,9 +141,16 @@ public static function create(
      * @param string $session_id The existing session id
      * @return RemoteWebDriver
      */
-    public static function createBySessionID($session_id, $selenium_server_url = 'http://localhost:4444/wd/hub')
+    public static function createBySessionID($session_id, $selenium_server_url = 'http://localhost:4444/wd/hub', $connection_timeout_in_ms = null, $request_timeout_in_ms = null)
```
Comment (OndraM) [inline lib/Remote/RemoteWebDriver.php]: Could you please update the phpDoc above to reflect the new params? It will be good to merge for me then!

**Comment by dmatora** [comment]
@OndraM done

File: lib/Remote/RemoteWebDriver.php (hunk 1 above)
Comment (dmatora) [reply lib/Remote/RemoteWebDriver.php]: Done

**Comment by OndraM** [comment]
Perfect, thanks!

**Pull request merged**